dao/src/main/java/com/epam/esm/dao/entity/Orders.java

```diff
@@ -0,0 +1,132 @@
+package com.epam.esm.dao.entity;
+
+import com.epam.esm.dao.entity.audit.AuditListener;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+@EntityListeners(value = AuditListener.class)
+public class Orders implements TableEntity {
```

Comment:
Orders? If you call this entity orders, I would expect to see array here

Reply:
We decide to rename Order to Orders in order to not get confused. Application already has Order in javax.persistence.criteria.